prado/src/main/java/com/schibsted/spain/prado/PradoGalleryActivity.kt

```diff
@@ -81,10 +84,16 @@ class PradoGalleryActivity : AppCompatActivity() {
   }
 
   private fun provideImageProvider() : ImageProvider {
-    when (imageProviderType) {
-      ImageProvider.ImageProviderType.PICASSO -> return PicassoImageProvider(this)
-      ImageProvider.ImageProviderType.GLIDE -> return GlideImageProvider()
+    try {
+      when (imageProviderType) {
+        ImageProvider.ImageProviderType.PICASSO -> return PicassoImageProvider(this)
+        ImageProvider.ImageProviderType.GLIDE -> return GlideImageProvider(this)
+      }
+    } catch (ipnae: ImageProviderNotAvailableError) {
+      toast("You have to provide at least one image provider (Glide or Picasso)")
```

Comment:
I would just crash here. Not having a supported image loader is not an acceptable state, so failing makes it obvious there is something here you should fix.